Source/Core/DolphinQt/Config/WiimoteControllersWidget.cpp

```diff
@@ -123,6 +126,25 @@ void WiimoteControllersWidget::CreateLayout()
   // Passthrough BT
   m_wiimote_layout->addWidget(m_wiimote_passthrough, m_wiimote_layout->rowCount(), 0, 1, -1);
 
+  int adapter_row = m_wiimote_layout->rowCount();
+  m_wiimote_layout->addWidget(m_wiimote_adapters_label, adapter_row, 1, 1, 1);
+  m_wiimote_layout->addWidget(m_wiimote_adapters, adapter_row, 2, 1, 2);
+  for (auto device : IOS::HLE::BluetoothRealDevice::ListDevices())
+  {
+    auto formatHex = [](int value) {
+      return QString::number(value, 16).toUpper().rightJustified(4, QChar::fromLatin1('0'));
+    };
+
+    QString deviceName = QString::fromStdString(device.name);
+    QString vid = formatHex(device.vid);
+    QString pid = formatHex(device.pid);
+
+    QString deviceInfo = QString::fromStdString("%1 (%2:%3)").arg(deviceName).arg(vid).arg(pid);
+
+    m_wiimote_adapters->addItem(deviceInfo);
+    m_wiimote_adapters->setItemData(m_wiimote_adapters->count() - 1, QVariant::fromValue(device));
+  }
+
```

Comment:
The currently configured device isn't being selected when you open the dialog. I think you need to add the `Config::Get` logic around here to make that happen.